server/einterfaces/mocks/ElasticsearchFixChannelIndexInterface.go

```diff
@@ -1,46 +0,0 @@
-// Code generated by mockery v2.23.2. DO NOT EDIT.
```

Comment:
These should not be deleted. Can you revert these changes?

Reply:
Those files are not generated by the actual or the new mock generation. Therefore I thought those where not needed anymore. I'll revert the commit !